### ISSUE TITLE
Add support for webpack v2

### ISSUE THREAD

**Issue by renarsvilnis** [nkt/atom-autocomplete-modules#38]
Hi,
Asking to add support for webpack 2.

As of Webpack 2 resolve config object combines `root`, `modulesDirectories`, `fallback` into `modules`  [[source]](https://gist.github.com/sokra/27b24881210b56bbaff7#resolving-options)

``` javascript
// webpack v1
{
  resolve: {
    moduleDirectories: ['node_modules'],
    root: path.resolve(__dirname, 'app')
  }
}

// webpack v2
{
  resolve: {
    modules: [
      path.resolve(__dirname, 'app'),
      'node_modules'
    ]
  }
}
```

I can try to make a PR for that, but wondering should I try to target the webpack version or just read in both version possible configs and combine them? As webpack v2 doesn't use v1 `resolve` object field names (`moduleDirectories`, `root`, ...) and vice versa.

Also noticed that there is no support autocompleting local files/modules, such as from `resolve.root`, adding Webpack v2 automatically adds support, should I add support for autocompleted `resolve.root` and other to Webpack v1?

**Comment by nkt** [comment]
I think we should support webpack v1 and v2 in fallback style. I mean:
```js
// webpack v2
if (Array.isArray(resolve.modules)) {
  return resolve.modules;
}

// webpack v1
const modules = Array.isArray(resolve.modules) ? resolve.modules : [];
if (resolve.root) {
  modules.unshift(resolve.root);
}

return modules;
```

Please create PR as you see fit, cause actually I don't using this functionality. Webpack support have been contributed by @pho3nixf1re, and I will be glad to see any contributions to this feature.

**Comment by pho3nixf1re** [comment]
@renarsvilnis I would be willing to help out if needed when you submit a PR to add this as well. I'm not using webpack v2 yet but will be soon enough that this interests me.

**Comment by renarsvilnis** [comment]
@pho3nixf1re I might need help validating my code.

How do you go about testing the package?
I'm trying to separate the webpack load functions into pure functions which could have unit tests.

**Comment by nkt** [comment]
Last weekend I started refactoring for adding support for other languages. I've started this package long time ago, but now feel the need of tests.

`use babel;` tells Atom to compile code using babel.

**Comment by jonyeezs** [comment]
Definitely needs restructuring and unit test.

But for this issue, it looks to be fixed by #64
